docker/compose.integration-tests.yaml

```diff
@@ -26,6 +26,7 @@ services:
 
   # test runner for integration tests in isolated environment
   test-runner:
+    platform: linux/amd64
```

Comment:
If we add these as well for the `nginx_logs_exporter`, `docker_stats_exporter`, `mail-test-target` and `celery-exporter` services as well, that should silence all the `The requested image's platform (linux/amd64) does blablabla` warnings I think.